packages/h5p-server/src/H5PEditor.ts

```diff
@@ -1112,7 +1123,7 @@ export default class H5PEditor {
             },
             libraryConfig: this.config.libraryConfig,
             postUserStatistics: false,
-            saveFreq: false,
+            saveFreq: this.config.contentUserStateSaveInterval,
```

Comment:
I guess the editor also needs the url where it can save states

Reply:
Yep. @otacke mentioned that the editor uses it to store data about tours.